docs/development_environment.mdx

```diff
@@ -59,6 +59,24 @@ If the Dev Container was set up correctly - it supports debugging by default, ou
 
 It is also possible to debug a remote Home Assistance instance (e.g., production instance) by following the procedure described [here](https://www.home-assistant.io/integrations/debugpy/).
 
+### Use devcontainer from WSL
+
+With WSL, you won't run into as many path issues because of the '/' and '\' differences between Linux and Windows. 
+This is handy when working on frontend code and using `script/develop`, since it's using stuff that works best on Linux.
+
+**Prerequisites**
+
+- [Docker](https://docs.docker.com/get-docker/)
```

Comment:
since [WSL2 supports systemd](https://devblogs.microsoft.com/commandline/systemd-support-is-now-available-in-wsl/) now, we should point the user both options for docker - [Docker Desktop](https://docs.docker.com/desktop/install/windows-install/) and [Docker-Engine](https://docs.docker.com/engine/install/) (_i prefer the installing the docker engine over docker desktop, since docker desktop comes with some unneeded overhead_)

Reply:
I didn't consider that option yet.
So far "Docker" in these instructions has meant "Docker desktop installed on windows".

I'm not sure if mentioning docker-engine here is helpful for most people. 
There is already a lot of information and I'm afraid adding more options makes it even more overwhelming.